src/olympia/hero/migrations/0007_auto_20200625_1139.py

```diff
@@ -0,0 +1,24 @@
+        ),
+        migrations.AlterField(
+            model_name='secondaryheromodule',
+            name='icon',
+            field=olympia.hero.models.WidgetCharField(choices=[('Extensions.svg', 'Extensions.svg'), ('Download.svg', 'Download.svg'), ('Login.svg', 'Login.svg'), ('Device Mobile.svg', 'Device Mobile.svg'), ('Secure.svg', 'Secure.svg'), ('Video Recorder.svg', 'Video Recorder.svg'), ('Popular.svg', 'Popular.svg'), ('Audio.svg', 'Audio.svg'), ('Mail.svg', 'Mail.svg'), ('Store Data.svg', 'Store Data.svg'), ('Globe.svg', 'Globe.svg'), ('Dictionaries.svg', 'Dictionaries.svg'), ('Play.svg', 'Play.svg'), ('Reader Mode.svg', 'Reader Mode.svg'), ('Midi.svg', 'Midi.svg'), ('Window New.svg', 'Window New.svg'), ('Private browsing.svg', 'Private browsing.svg'), ('Help.svg', 'Help.svg'), ('Search.svg', 'Search.svg'), ('Performance.svg', 'Performance.svg'), ('Highlights.svg', 'Highlights.svg'), ('Developer.svg', 'Developer.svg'), ('History.svg', 'History.svg'), ('Themes.svg', 'Themes.svg'), ('Subscribe.svg', 'Subscribe.svg'), ('Accessibility.svg', 'Accessibility.svg'), ('Pin.svg', 'Pin.svg'), ('Sync.svg', 'Sync.svg'), ('Save.svg', 'Save.svg'), ('Customize.svg', 'Customize.svg'), ('Tracking Protection.svg', 'Tracking Protection.svg'), ('Bookmark Outline.svg', 'Bookmark Outline.svg'), ('Notification.svg', 'Notification.svg'), ('Folder.svg', 'Folder.svg'), ('Bookmark.svg', 'Bookmark.svg'), ('Tab new.svg', 'Tab new.svg'), ('Trackers Disabled.svg', 'Trackers Disabled.svg')], max_length=255),
+        ),
+    ]
```

Comment:
can we drop this migration file?  Lisa's making changes to it and I don't want to add extra complications.  (Plus when it's done, both of those fields will be gone anyway)

Reply:
https://github.com/mozilla/addons-server/pull/14727/commits/7663e3a3213080fb1d5d36e9eb3331998231fce2